### PR TITLE
Add example of BigInt-to-string formatting

### PR DESCRIPTION
**Pull request by sffc**

### CODE REVIEW

**Comment by sffc** [comment]
I also added a couple paragraphs editorializing about why we are using strings, and how we don't intend for strings to set precedent for use elsewhere.

File: README.md [hunk 1]
```diff
@@ -176,7 +176,22 @@ nf.format(string);
...
+    style: "currency",
+    currency: "EUR",
+});
+const bi = 1000000000000000110000n;
+nf.format(bi + "E-6");
+// Current:  "€1,000,000,000,000,000.10"
+// Proposed: "€1,000,000,000,000,000.11"
+```
+
+The general syntax for decimal strings, essentially `#.#E#`, is a widely understood interchange format in computing. The specific version we use is the ECMA-262 `StringNumericLiteral` grammar, which also allows non-base-10 numbers like hexadecimal and binary.
```
Comment (syg) [inline README.md:192]: I don't know if I would call the E-notation `#.#E#` an interchange format, but I do agree it's widely understood notation.

**Review by syg — APPROVED** [review]
lgtm! Thanks for the note at the bottom about numeric strings being intended for use as pass-through.

File: README.md (hunk 1 above)
Comment (sffc) [reply README.md]: Sure, I removed the word "interchange".

**Pull request merged**